Bubble_Sort/bubble_sort.ts

```diff
@@ -0,0 +1,44 @@
+//Bubble Sort In Typescript
+
+function bubble(array : number[]) : number[]
+{
+ console.log("Original Array : " + array); // print original array
+ console.log("Steps in bubble sorting the array : ");
+    
+     //FOR LOOP to perform bubble sort(swapping)
+     for( let i = 0; i < array.length ; i++)
+     { 
+          for(let j = 0; j < array.length - 1; j++ ) 
+          {
+               if(array[j] > array[j + 1])
+               {
+                    let swap = array[j];
+                    array[j] = array[j + 1];
+                    array[j + 1] = swap;
+                   // prints arrray to check after every bubble sort step 
+                   console.log(array); 
+            }
+        }
+    }
+// print bubble sorted array
+console.log(" Final Array : " + array); 
+return array;
+}
+  
+// consider the array input of bubble sort (may choose any array)
+bubble([2, 1, 99, 8, 4, 0, 18]);
+
+// OUTPUT
```

Comment:
Please change it to multi line comments instead of so many single line comments.

Reply:
unchecked.